### PR TITLE
Implemented service GET entity types w/o details param

### PR DESCRIPTION
**Pull request by joaoGabriel55**

### CODE REVIEW

**Comment by joaoGabriel55** [comment]
- [x] Implement GET entity types service w/o details param 
- [ ] Implement GET entity types service with details param
- [ ] Implement GET entity types service by specific type, like Vehicle

File: test/functionalTest/cases/0000_ngsild/ngsild_entity_types.test [hunk 1]
```diff
@@ -239,18 +251,24 @@ Date: REGEX(.*)
...
-    "T1",
-    "T2",
-    "T3",
-    "T4"
-]
+{
+    "id": "urn:ngsi-ld:EntityTypeList:REGEX(.*)",
+    "type": "EntityTypeList",
+    "typeList": [
+        [
```
Comment (kzangeli) [inline test/functionalTest/cases/0000_ngsild/ngsild_entity_types.test]: Why a double array?

The section 5.2.24 of the NGSI-LD API spec says that typeList is a array of strings, not an array of arrays of strings ...

File: src/lib/orionld/db/dbEntityTypesGet.cpp [hunk 2]
```diff
@@ -176,5 +177,19 @@ KjNode* dbEntityTypesGet(OrionldProblemDetails* pdP)
...
+  KjNode* typeNodeResponseP = kjObject(orionldState.kjsonP, NULL);
+  KjNode* idNodeP           = kjString(orionldState.kjsonP, "id", entityTypesId);
+  KjNode* typeNodeP         = kjString(orionldState.kjsonP, "type", "EntityTypeList");
+  KjNode* typeNodeListP     = kjArray(orionldState.kjsonP,  "typeList");
+
+  kjChildAdd(typeNodeListP, sortedArrayP);
+  kjChildAdd(typeNodeResponseP, idNodeP);
+  kjChildAdd(typeNodeResponseP, typeNodeP);
+  kjChildAdd(typeNodeResponseP, typeNodeListP);
+
```
Comment (kzangeli) [inline src/lib/orionld/db/dbEntityTypesGet.cpp:191]: This is weird ... I don't see the usage of "sortedArrayP" ...
How is "typeNodeListP" filled in?

Perhaps you should remove the creation and addition of typeNodeListP to typeNodeResponseP and just add sortedArrayP to typeNodeResponseP?

Comment (kzangeli) [reply src/lib/orionld/db/dbEntityTypesGet.cpp:191]: Fixed in 068159b18

File: test/functionalTest/cases/0000_ngsild/ngsild_entity_types.test (hunk 1 above)
Comment (kzangeli) [reply test/functionalTest/cases/0000_ngsild/ngsild_entity_types.test]: Fixed in 068159b

**Pull request merged**